CodeConcierge/Reviews-module
# Basic client/server/webpack

Test instructions:

clone this Reviews Repo 
make sure you have webpack and nodemon installed globally 
enter the repo in terminal. 
use npm run build
use npm start
insert localhost:3004 into a web browser 

picture of working server and client

<img width="1343" alt="screen shot 2019-02-08 at 3 57 23 pm" src="https://user-images.githubusercontent.com/40609789/52512757-7bf45680-2bbb-11e9-86ff-e57af545a4b5.png">
